app/components/embed/download/file_list_item_component.html.erb

```diff
@@ -1,3 +1,8 @@
+<% if show_headers %>
```

Comment:
Can we combine these two conditionals into one?  Can they be put into a method on the class (with documentation about why we are showing this and in what situations)?